### PR TITLE
Remove staticPodPath for workers

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/4447

### CODE REVIEW

File: v_4_0_0/files/config/kubelet-config-worker.yaml.tmpl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+evictionSoft:
+  memory.available: "500Mi"
+evictionHard:
+  memory.available: "200Mi"
+evictionSoftGracePeriod:
+  memory.available: "5s"
+evictionMaxPodGracePeriod: 60
+authentication:
+  anonymous:
+    enabled: true # Defaults to false as of 1.10
```
Comment (xh3b4sd) [inline v_4_0_0/files/config/kubelet-config-worker.yaml.tmpl]: I see this is copy/paste but can we maybe have comments above as a usual block? This here makes it harder to read the actual YAML structure.

**Comment by kopiczko** [comment]
I'd also prefer to have an `{{ if }}`.

**Comment by corest** [comment]
Well, yes, I would also love to have. There is just no data for statement. Anyway, I moved it to the flag option from config and add it only in master template

**Comment by kopiczko** [comment]
Isn't the flag deprecated?

**Comment by corest** [comment]
Nope, it was moved into the config just to make template clearer

**Pull request merged**